app/src/main/java/com/fabulouszanna/pokedex/PokemonApp.kt

```diff
@@ -2,13 +2,13 @@ package com.fabulouszanna.pokedex
 
 import android.app.Application
 import com.fabulouszanna.pokedex.repo.MainDatabase
-import com.fabulouszanna.pokedex.repo.pokemon.SinglePokemonViewModel
+import com.fabulouszanna.pokedex.ui.pokemonlist.SinglePokemonViewModel
 import com.fabulouszanna.pokedex.repo.ability.AbilityRepository
-import com.fabulouszanna.pokedex.repo.ability.AbilityViewModel
+import com.fabulouszanna.pokedex.ui.ability.AbilityViewModel
 import com.fabulouszanna.pokedex.repo.pokemon.PokemonRepository
-import com.fabulouszanna.pokedex.repo.pokemon.PokemonViewModel
+import com.fabulouszanna.pokedex.ui.pokemonlist.PokemonViewModel
```

Comment:
You could rename it to follow the pattern
So `PokemonViewModel` becomes `PokemonListViewModel`

Reply:
Addressed in next commit